### ISSUE TITLE
Segmentation fault with v1.3.2b but not older

### ISSUE THREAD

**Issue by amit491** [gpertea/stringtie#93]
Hi,
I have been using StringTie for quite sometime now on human unstranded RNA-seq data with good results. I have now some **Stranded** RNA-seq data for which I am excited to see the new params. in the latest release. However when I try to run using the Linux binary, I immediately get this error -

```
[01/27 12:55:01] Loading reference annotation (guides)..
Segmentation fault

```
I have tried two different sources of BAM now, both STAR aligned, but one generated by me and the other at an external facility. I get the same error with v.1.3.2b irrespective of whether I use the strand parameter or not, or whether I use threads (-p) or not.

Whereas the prev. ver. binary (1.3.1c) is working fine. But here I do not have the option to specify strandedness.
Please let me know if there is any potential fix for this.
Many thanks

**Comment by gpertea** [comment]
Could be a problem with loading the annotation file you're using there.. (perhaps a new bug due to the recent changes in the GFF parser?). Could you please share that annotation file with me?

**Comment by amit491** [comment]
Thanks. I have used the latest rel. of Ensembl.
ftp://ftp.ensembl.org/pub/grch37/release-87/gtf/homo_sapiens/Homo_sapiens.GRCh37.87.gtf.gz
ftp://ftp.ensembl.org/pub/release-87/gtf/homo_sapiens/Homo_sapiens.GRCh38.87.gtf.gz

So, the prev. version is working with both the GRCh37 and GRCh38 releases of the Ensembl87.
And the latest has the same error with either of the above GTFs.

**Comment by gpertea** [comment]
Thanks -- I just confirmed that the GTF parser is now crashing on those files. A fix should be coming along soon.

**Comment by amit491** [comment]
Many many thanks! Looking forward to the release 👍

**Comment by gpertea** [comment]
This is now fixed, sorry about that silly error..  Please download the fixed version (1.3.2c) from:
 http://ccb.jhu.edu/software/stringtie/#install

**Comment by amit491** [comment]
Thanks again. Its working now perfectly!
